README.md

```diff
@@ -17,6 +17,53 @@ Text input mask for React Native on iOS and Android.
 
 ```bash
 npm install --save react-native-text-input-mask
+# --- or ---
+yarn add react-native-permissions
+```
+
+#Installation
+
+<details>
+  <summary><h4>For RN >= 0.60</h4></summary>
+
+#### iOS
+
+1. In XCode, in the project navigator, right click your `[your project's name]` folder, choose ➜ `Add Files to [your project's name]`
+
+![Create Swift File](https://i.imgur.com/00K5UZ1.png)
+
+2. Select `Swift File` ➜ `Next`
+
+![Create Swift File](https://i.imgur.com/Mdc9MLk.png)
+
+3. Specify name for example `Dummy.swift` ➜ `Create`
+
+![Create Swift File](https://i.imgur.com/2HSk7Jp.png)
+
+4. Now a pop up is shown select `Create Bridging Header`
+
+![Create Swift File](https://i.imgur.com/f2zA0n9.png)
```

Comment:
Do we need all this previous steps before the podfile adding?

Reply:
Yes we have to do that before `pod install` otherwise pod install will fail with error that `no swift version is specified`. These steps will enable swift integration and remove that error.

Reply:
if you use use_framworks! in the podspec that does not get fixed, or adding the swift version to the podspec

Reply:
use_frameworks! does not fix that and also it does not work with 0.60.* versions.
While specifying swift version in `RNInputMask` podspec like `spec.swift_version = '4.2'` also is not useful.
As i got error library not found error. Here is another library mentioned that.
https://react-native-kit.github.io/react-native-track-player/install/#troubleshooting
i have got the same error.

Reply:
@acollazomayer Fixed now. Added instructions for `react-native >= 0.61` as it supports `use_frameworks!`. For `RN = 0.60.*` people need to add that `Dummy.swift` as `use_framework!` does not work with it.